tests/returntypes/test_autograd_qnode_shot_vector.py

```diff
@@ -0,0 +1,687 @@
+# Copyright 2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Integration tests for using the jax interface with shot vectors and with a QNode"""
+
+import pytest
+
+import pennylane as qml
+from pennylane import numpy as np
+from pennylane import qnode
+
+pytestmark = pytest.mark.autograd
+
+shots = [((5, 2), 1, 10)]
```

Comment:
Let's have another item where the tuple is not in the first position.
```suggestion
shots = [((5, 2), 1, 10), (1, 10, (5, 2))]
```